src/main.rs

```diff
@@ -122,3 +148,58 @@ mod tests {
         assert_eq!(segment_time_to_srt_time_string(-4550), "00:00:00,000");
     }
 }
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
What happened here?

Reply:
EMPTY LINES GEORGE!

Reply:
This should probably be in draft still. It's not working fully, and I want to understand it before merging.

Reply:
Seems you're falling asleep at the keyboard again.